docker/run_setup_postgres.sh

```diff
@@ -6,35 +6,32 @@
 
 set -e
 
-# Sets up postgres tables, stored procedures, types, and such.
+# Sets up postgres tables.
 
 # NOTE: This should only be run if you want to drop an existing database and
 # create a new one from scratch for a development environment.
 
 cd /app
 
 # This is a webapp environment variable. If it doesn't exist, then use
-# "breakpad"
+# "breakpad".
 DATABASE="${DATASERVICE_DATABASE_NAME:-breakpad}"
 
 # Wait until postgres is listening
 urlwait "${DATABASE_URL}" 10
 
-# This drops and re-creates the db; the --dropdb code will prompt the user
-# before it does anything giving the user a chance to do a "oh no--don't do
-# that!" kind of thing
-echo "Dropping existing db and creating new db named (${DATABASE})..."
-./socorro/external/postgresql/setupdb_app.py \
-                  --database_name="${DATABASE}" \
-                  --dropdb \
-                  --createdb
+# Create the database if it doesn't exist
+echo "Dropping and creating db..."
+python socorro/scripts/db.py drop || true
+python socorro/scripts/db.py create || true
```

Comment:
The `|| true` bit here is to ignore exit codes. It's ok if it can't drop a database that doesn't exist.